autogen.sh

```diff
@@ -71,8 +83,8 @@ aclocal -I config || bail_out
 # --- Step 2:
 
 echo "Running libtoolize"
```

Comment:
Here you could use the macro `LIBTOOLIZE` to give a more precise information:

    echo "Running $LIBTOOLIZE"